client_linux_e2e_test.go

```diff
@@ -0,0 +1,57 @@
+//go:build integration
```

Comment:
I kinda prefer passing an envvar or extra flag these days vs a build tag for discoverability. Then you can `t.Skip` when those aren't present to make it obvious the test exists.